specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -278,9 +279,9 @@ def vanishing_polynomialcoeff(xs: Sequence[BLSFieldElement]) -> PolynomialCoeff:
     """
     Compute the vanishing polynomial on ``xs`` (in coefficient form)
     """
-    p = [1]
+    p = PolynomialCoeff([1])
     for x in xs:
-        p = multiply_polynomialcoeff(p, [-int(x), 1])
+        p = multiply_polynomialcoeff(p, PolynomialCoeff([-int(x), 1]))
```

Comment:
After d5b4e6cc6fc66b60882f1fcb29892f2fcfc1ab3c fix, we now have a error here:
```
eth2spec/eip7594/minimal.py:5014: in vanishing_polynomialcoeff
    p = multiply_polynomialcoeff(p, PolynomialCoeff([-int(x), 1]))
/Users/hww/.pyenv/versions/pypy3.9-arm64/envs/pypy39-202209/lib/pypy3.9/site-packages/remerkleable/complex.py:289: in __new__
    input_views.append(elem_cls.coerce_view(el))
/Users/hww/.pyenv/versions/pypy3.9-arm64/envs/pypy39-202209/lib/pypy3.9/site-packages/remerkleable/basic.py:192: in coerce_view
    return cls(v)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <class 'eth2spec.eip7594.minimal.BLSFieldElement'>, value = -52435875175126190...3658699938581184512

    def __new__(cls, value: int):
        if value < 0:
>           raise ValueError(f"unsigned type {cls} must not be negative")
E           ValueError: unsigned type <class 'eth2spec.eip7594.minimal.BLSFieldElement'> must not be negative

/Users/hww/.pyenv/versions/pypy3.9-arm64/envs/pypy39-202209/lib/pypy3.9/site-packages/remerkleable/basic.py:81: ValueError
```

Reply:
The fix is to do `BLS_MODULUS - int(x)` so the value is in the correct range -- I'm having trouble running `make test`, so I'm gonna push a blind fix for now